packages/api/src/utils/constants.ts

```diff
@@ -14,4 +14,7 @@ const clientEndpoints = {
   passwordMagicLink: '/passwordRecovery/magicLink',
   registrationMagicLink: '/registration',
 };
-export default { authTokens, clientEndpoints };
+
+const roles_key = 'roles';
```

Comment:
If you create js variable with underscore, make sure its uppercased const, otherwise it looks like C++ :P